README.md

```diff
@@ -107,14 +107,14 @@ pip install pandas
 See the [full installation instructions](https://pandas.pydata.org/pandas-docs/stable/install.html#dependencies) for minimum supported versions of required, recommended and optional dependencies.
 
 ## Installation from sources
-To install pandas from source you need [Cython](https://cython.org/) in addition to the normal
+To install pandas the from source you need [Cython](https://cython.org/) in addition to the normal
 dependencies above. Cython can be installed from PyPI:
 
 ```sh
 pip install cython
 ```
 
-In the `pandas` directory (same one where you found this file after
+In the `pandas` directory (the same one where you found this file after
```

Comment:
this change is slightly better, but since in parentheses it doesn't add much to understanding.